shell/app/manifests.cc

```diff
@@ -43,13 +23,3 @@ const service_manager::Manifest& GetElectronContentBrowserOverlayManifest() {
           .Build()};
   return *manifest;
 }
```

Comment:
`.RequireCapability("pdf_compositor", "compositor")` should be removed.

Reviewing on wrong line here because of github UI :(